### PR TITLE
Add max packet overwarp & tachyon cloak reduction to rules

### LINKED ISSUE #422: A Packet Physics starbase two warp 13 flingers can fling at warp 17
I have confirmed that it is limited to 16 in OG stars.

### PR DESCRIPTION
**Pull request by Bertie690**
Fixes #422

Did not get to test this as my local instance is being grumpy as usual.
(I hope I added the rules to the schema files correctly...)

### CODE REVIEW

File: cs/rules.go [hunk 1]
```diff
@@ -285,6 +287,7 @@ func NewRulesWithSeed(seed int64) Rules {
 			WormholeMinPlanetDistance: 30,
 		},
 		TachyonCloakReduction:            5,
+		TachyonMaxCloakReduction:         19, // cloak reduction caps at (100 - 19)% or 81%
```
Comment (sirgwain) [inline cs/rules.go]: We should store this as a human readable value, so use 81, not 19. Don't make the player do math in their head if they are setting this as a custom rule.

File: db/schema/games/0001_schema.up.sql [hunk 2]
```diff
@@ -92,6 +92,7 @@ CREATE TABLE IF NOT EXISTS rules (
   terraformCost TEXT,
   starbaseComponentCostFactor REAL,
   packetDecayRate TEXT,
```
Comment (sirgwain) [inline db/schema/games/0001_schema.up.sql]: we aren't storing rules in the database yet. We'll need to blow away this rules table when we eventually do. Can you revert this change?

(also, the way the schema stuff works is you create new files, because the old ones have already been applied. You never modify the existing schema files)

Comment (Bertie690) [reply db/schema/games/0001_schema.up.sql]: Do I also revert `wasm\wasm\generated.go`?

Comment (sirgwain) [reply db/schema/games/0001_schema.up.sql]: nah, that's needed. You should run the `go generate ./...` command if you change any models.

**Review by sirgwain — APPROVED** [review]
